server/config/conditions.ts

```diff
@@ -75,7 +75,7 @@ export default {
       code: '5db26ab3-9b6f-4bee-b2aa-53aa3f3be7dd',
       category: 'Residence at a specific place',
       text: 'You must reside within [INSERT REGION] while of no fixed abode, unless otherwise approved by your supervising officer.',
```

Comment:
The text which shows on the screens should match what is on the resultant licence maybe?

Reply:
yes, it should. It altered the mean when selecting from drop down, so I'll update this to include `the [value] probation region`

Reply:
Pushed now.